FHEM/00_SIGNALduino.pm

```diff
@@ -3827,8 +3848,27 @@ sub SetSens {
 }
 
 
+############################################################################
+########## Section & cc1101 settings for manufacturer specific´s  ##########
+package cc1101_manspecific;
+
+sub Register {
+	my ($aVal) = @_;
```

Comment:
Hmm eine Liste in einen scalar schreiben. Was kommt da heraus?

Reply:
Die Übernahme ist natürlich via C&P, daher muss es nicht unbedingt sein.